Pipfile

```diff
@@ -16,6 +16,7 @@ pandas = "==2.2.0"
 numpy = "==1.26.3"
 dash-iconify = "*"
 scipy = "==1.12.0"
+colorcet = "<4,>=3.0.1"
```

Comment:
please remove this dependency and simply add the color palette as list of hex colors